config/jobs/kubernetes-sigs/kernel-module-management/kernel-module-management-presubmits.yaml

```diff
@@ -12,6 +12,37 @@ presubmits:
       - image: golang:1.18
         command:
         - ci/prow/build
+  - name: pull-kernel-module-management-check-api-changes
+    always_run: true
+    decorate: true
```

Comment:
Should we start with optional so we don't block PRs until we see the job is stable?

Reply:
I ran the job locally and it worked. Let's take the risk 🙂